static/css/diff.less

```diff
@@ -115,4 +115,34 @@ ins {
 del {
   color: #e44028;
   text-decoration: line-through;
+}
+#back2Top {
+    width: 40px;
+    line-height: 40px;
+    overflow: hidden;
+    z-index: 999;
+    display: none;
+    cursor: pointer;
+    -moz-transform: rotate(270deg);
+    -webkit-transform: rotate(270deg);
+    -o-transform: rotate(270deg);
+    -ms-transform: rotate(270deg);
+    transform: rotate(270deg);
+    position: fixed;
+    bottom: 50px;
+    right: 0;
+    background-color: #DDD;
+    color: #555;
```

Comment:
not sure what LESS variable this is off the top of my head, but I'm sure one exists in colors.less!

Reply:
I have removed this no need of it.

Reply:
Since the color was changed so I had to put it!